src/client/network.ts

```diff
@@ -421,13 +422,8 @@ export class Response extends ChannelOwner<channels.ResponseChannel, channels.Re
     return { ...this._headers };
   }
 
-  async finished(): Promise<Error | null> {
-    return this._wrapApiCall(async (channel: channels.ResponseChannel) => {
-      const result = await channel.finished();
-      if (result.error)
-        return new Error(result.error);
```

Comment:
I don't see where we reject `this._finishedPromise`. Should we throw from somewhere? Perhaps we miss a test?

Reply:
We did not and we should not...